### DISCUSSION TITLE
CLI to start Medusa Recipe Projects

### DISCUSSION THREAD

**Discussion by greendesertsnow** [medusajs/medusa#7041] [Feature Requests]
Hey!
Would you guys please modify the CLI so it's a easy procedure to start a subscription based project or a multivendor marketplace, or a multiregion store without touching the Type ORM...
Your recipes are too hard to follow for beginners. Too many things to learn and mess up at once. 
We need CLI to manage that.

https://docs.medusajs.com/recipes

**Comment by muhammadx2k** [comment]
Yes, we need better documentation or easier setup for these recipes. I am having trouble building a multivendor marketplace with subscription-based products.
